build.py

```diff
@@ -87,7 +87,9 @@
         '22.01',  # upstream container
         '1.10.0',  # ORT
         '2021.2.200',  # ORT OpenVINO
-        (('2021.2', None), ('2021.4', '2021.4.582')),  # Standalone OpenVINO
+        (('2021.2', None),
+         ('2021.4', '2021.4.582'),
+         ('SPECIFIC','f2f281e6')),  # Standalone OpenVINO
```

Comment:
Add comment above describing use of "SPECIFIC"

Reply:
Can we use COMMIT_SHA instead of specific to make it more accurate?

Reply:
I think the first value is meant to be some version number, `COMMIT_SHA` doesn't fit that well

Reply:
Then maybe just adding a commet like David G said is enough